### PR TITLE
Userstory105 student confirms profile info web portion

### PR DESCRIPTION
**Pull request by 2011sandeepraj**

### CODE REVIEW

File: sources/packages/api/src/route-controllers/student/models/student.dto.ts [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
+  fullName: string;
+  addressLine1: string;
+  addressLine2: string;
+  city: string;
+  provinceState: string;
+  country: string;
+  postalCode: string;
+  gender: string;
+  phoneNumber: string;
+  dateOfBirth: string;
```
Comment (2011sandeepraj) [inline sources/packages/api/src/route-controllers/student/models/student.dto.ts:12]: Take the formatting to front end. Pass this back as a date

File: sources/packages/web/src/components/fa-application/ConfirmStudentAidBCProfileInfo.vue [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        <label for="gender">Gender</label>
+        <div class="p-field p-col">{{ studentConfirmInfo.gender }}</div>
+      </div>
+    </div>
+    <h4 class="p-mb-5">If the Information is incorrect, please click here</h4>
+    <div class="p-fluid p-formgrid p-grid">
+      <div class="p-field p-col-4">
+        <label for="homeAddress">Home Address</label>
+        <!-- <div class="p-field p-col">
+          #112 1042 Jervis Street,
```
Comment (2011sandeepraj) [inline sources/packages/web/src/components/fa-application/ConfirmStudentAidBCProfileInfo.vue:29]: Remove commented code

File: sources/packages/api/src/helpers/utilfunctions.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { StudentConfirmInfo } from "../route-controllers/student/models/student.dto";
+export default class Helper {
+  static formatDate(date: Date): string {
+    var options = { year: "numeric", month: "long", day: "numeric" };
```
Comment (2011sandeepraj) [inline sources/packages/api/src/helpers/utilfunctions.ts:4]: Take to front end

**Review by fwpushan — APPROVED** [review]
LGTM. Please fix those co-reviewed items in another PR.

**Pull request merged**